app/src/main/java/com/keylesspalace/tusky/network/MastodonApi.kt

```diff
@@ -656,4 +657,13 @@ interface MastodonApi {
         @Header("Authorization") auth: String,
         @Header(DOMAIN_HEADER) domain: String,
     ): NetworkResult<ResponseBody>
+
+    @GET("api/v1/tags/{name}")
+    fun tag(@Path("name") name: String): Single<HashTag>
+
+    @POST("api/v1/tags/{name}/follow")
+    fun followTag(@Path("name") name: String): Single<HashTag>
+
+    @POST("api/v1/tags/{name}/unfollow")
+    fun unfollowTag(@Path("name") name: String): Single<HashTag>
```

Comment:
make these `suspend fun call(...): NetworkResult<Hashtag>` and then call them like so:
```
mastodonApi.call(...)
    .fold(
        { tag ->
            ...     
        },
       { throwable ->
           ...      
       }
)
```

Reply:
Oh sorry, I looked at the wrong sample